src/test/java/seedu/address/logic/commands/AddWeightCommandTest.java

```diff
@@ -0,0 +1,53 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.FitNus;
+import seedu.address.model.ReadOnlyFitNus;
+import seedu.address.model.person.Weight;
+import seedu.address.testutil.WeightBuilder;
+
+public class AddWeightCommandTest {
```

Comment:
Similar comments as above. Consider more test cases. A good idea would be to run the tests with coverage in intellij to see possible lines that you missed out in AddWeightCommand and AddHeightCommand. Week 8 topics has a video on how to run with coverage if you are unsure.